### PR TITLE
Framework: updated container gcc openmpi build for github

### PR DESCRIPTION
**Pull request by fryeguy52**
@trilinos/framework 

## Motivation
This updates the containerized build for the new github actions gcc 10.3 openmpi 4.1.6 environment

### CODE REVIEW

File: packages/framework/ini-files/config-specs.ini [hunk 1]
```diff
@@ -2935,6 +2935,11 @@ use PACKAGE-ENABLES|NO-PACKAGE-ENABLES
 
 use COMMON_SPACK_TPLS
 
+opt-set-cmake-var TPL_BLAS_LIBRARY_DIRS     STRING FORCE : ${OPENBLAS_ROOT|ENV}/lib
+opt-set-cmake-var TPL_BLAS_LIBRARIES        STRING FORCE : ${OPENBLAS_ROOT|ENV}/lib/libopenblas.a;-L${OPENBLAS_ROOT|ENV}/lib;-lgfortran;-lgomp;-lm
+opt-set-cmake-var TPL_LAPACK_LIBRARY_DIRS   STRING FORCE : ${OPENBLAS_ROOT|ENV}/lib
+opt-set-cmake-var TPL_LAPACK_LIBRARIES      STRING FORCE : ${OPENBLAS_ROOT|ENV}/lib/libopenblas.a;-L${OPENBLAS_ROOT|ENV}/lib;-lgfortran;-lgomp;-lm
```
Comment (sebrowne) [inline packages/framework/ini-files/config-specs.ini:2941]: We could get around this if we set a more-general variable in the container, correct?  Meaning, if openblas' module set `BLAS_ROOT` instead, we could probably not need these?

**Review by sebrowne — DISMISSED** [review]

**Comment by sebrowne** [comment]
Hit by same failure as #12802

File: packages/framework/ini-files/config-specs.ini (hunk 1 above)
Comment (fryeguy52) [reply packages/framework/ini-files/config-specs.ini:2941]: yes we could do something like that. The other thing we could do is add links in the bin directory of openblas install so you get libblas and liblapack as links pointing to libopenblas

Comment (sebrowne) [reply packages/framework/ini-files/config-specs.ini:2941]: Spoke with Joe at standup today and decided to leave it as is for now.

**Review by sebrowne — APPROVED** [review]

**Pull request merged**